internal/fields/dependency_manager.go

```diff
@@ -221,5 +229,21 @@ func transformImportedField(fd FieldDefinition) common.MapStr {
 		}
 		m.Put("fields", t)
 	}
+
+	if len(fd.MultiFields) > 0 {
+		var t []common.MapStr
+		for _, f := range fd.MultiFields {
+			i := transformImportedMultiField(f)
```

Comment:
nit: an alternative implementation - you can introduce a new interface `IFieldDefinition` with `GetName`, `GetDescription`, `GetType`, ..., so that you can unify the transformation - single function `transformImportedField`.